accounts/cachedb_test.go

```diff
@@ -313,6 +315,7 @@ func TestCacheFind_CacheFind(t *testing.T) {
 			continue
 		}
 	}
+	cache.close()
```

Comment:
Ditto as in https://github.com/ethereumproject/go-ethereum/pull/250/files#diff-f2f6a0212b87040314d94add9102b28fR236